robber/matchers/called.py

```diff
@@ -0,0 +1,32 @@
+from robber import expect
+from robber.matchers.base import Base
+
+
+class Called(Base):
+    """
+    expect(function).to.be.called()
+    """
+
+    def matches(self):
```

Comment:
Takes a look at how the `Exception` matcher is implemented (https://github.com/EastAgile/robber.py/blob/master/robber/matchers/exception.py#L10). You need to check that `self.actual` is actually an instance of `Mock`, otherwise, return a proper error message.

Similarly, fix the same issue in `CalledOnce` and add necessary failure tests for both of them.